explore/src/main/scala/explore/config/ConfigurationPanel.scala

```diff
@@ -149,26 +150,16 @@ object ConfigurationPanel {
     )
 
   // TODO Move these to react-datetime
-  implicit class LocalDateTimeOps(val localDate: LocalDateTime) extends AnyVal {
-    def toJsDate: js.Date =
-      new js.Date(
-        localDate.getYear,
-        localDate.getMonthValue - 1,
-        localDate.getDayOfMonth,
-        localDate.getHour(),
-        localDate.getMinute()
-      )
+  implicit class InstantOps(val instant: Instant) extends AnyVal {
+    // DatePicker only works in local timezone, so we trick it by adding the timezone offset.
+    def toDatePickerJsDate: js.Date =
+      new js.Date(instant.toEpochMilli.toDouble + (new js.Date()).getTimezoneOffset() * 60000)
```

Comment:
is `js.Date` always local?
Have you seen this?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC

Reply:
Yup, I tried that first.

`js.Date` just stores millis since epoch, but the issue here is `DatePicker`, which will only show it as local time. See: https://github.com/Hacker0x01/react-datepicker/issues/1787 (open for almost 3 years and still unsolved).